src/stores/businessStore.ts

```diff
@@ -218,6 +224,10 @@ export const useBusinessStore = defineStore('business', {
       this.businessInfo.foundingDate = val
     },
 
+    setStartDate (val: string) {
+      this.businessInfo.startDate = val
```

Comment:
For clarity, type of val should be `ApiDateTimeUtc`.

Also, don't you need to call this in App.vue when the business data is loaded?

Reply:
For first,  I will change it
For second, no do not need to call it. This will be set in store when `businessInfo` is set (`fetchBusiness` api call)